services/comments_service.py

```
@@ -1,34 +1,87 @@
-from datetime import datetime
-from models.comment import Comment, CommentInternal, CommentResponse
-from models.post import Post
+from datetime import datetime,timezone
+import uuid
+
+from models.comment import Comment, CommentResponse
 from repositories.comments_repo import get_comments, save_comments
 from fastapi import HTTPException
+from typing import Annotated, List, Optional
```

Ruff (0.14.13)

`typing.List` is deprecated, use `list` instead

(UP035)